### ISSUE TITLE
IDF Editor, Swimming pool: Indoor / Surface Name selection

### ISSUE THREAD

**Issue by zhukovgreen** [NREL/EnergyPlus#4844]
Once you click to select a surface, the dialog refer you on Constructions instead of Surfaces
![image](https://cloud.githubusercontent.com/assets/10520003/7045548/6053e304-ddff-11e4-8dfc-3d3e79281918.png)

**Comment by mjwitte** [comment]
@ZhukovGreen Thanks for noting this.  This should reference any surface that can be a floor (BuildingSurface:Detailed, Floor:*).  A quick workaround is to edit the swimming pool IDD entry.

Current:

```
SwimmingPool:Indoor,
  A1,  \field Name
       \required-field
       \type alpha
  A2,  \field Surface Name
       \required-field
       \note To be matched with a construction in this input file
       \type object-list
       \object-list ConstructionNames
       \reference SurfaceNames
       \reference SurfAndSubSurfNames
       \reference AllHeatTranSurfNames
       \reference HeatTranBaseSurfNames
       \reference OutFaceEnvNames
       \reference AllHeatTranAngFacNames
       \reference RadGroupAndSurfNames
       \reference SurfGroupAndHTSurfNames
       \reference AllShadingAndHTSurfNames
```

Workaround:

```
SwimmingPool:Indoor,
  A1,  \field Name
       \required-field
       \type alpha
  A2,  \field Surface Name
       \required-field
       \note To be matched with a floor surface in this input file
       \type object-list
       \object-list RadiantSurfaceNames
```

The final fix will be different, but this will work for now.

**Comment by zhukovgreen** [comment]
Thank you, Michael.

**Comment by mjwitte** [comment]
@rglazer2 This objectlist should be limited to floors only, so this needs a new \reference FloorSurfaceNames.  Add this to 
BuildingSurface:Detailed, Floor:Detailed, Floor:GroundContact, Floor:Adiabatic, and Floor:Interzone.

**Comment by mjwitte** [comment]
Closed via #5078

**Comment by BD-Swimming-Pool** [comment]
Now easy to making a swimming pool. For details
http://bdswimmingpool.com/
